packages/babel-plugin-rn-stylename-to-style/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -1229,6 +1229,85 @@ function Test({ style, active, submit, disabled, titleStyle }) {
 }
 
 
```

Comment:
Please ignore this file change. It is not related to this PR. Just an extra test for the CSS parts compilation which I had lying around in my master not commited.